demos/local-only-todolist/lib/models/schema.dart

```diff
@@ -0,0 +1,98 @@
+import 'package:powersync/powersync.dart';
+import 'package:powersync_flutter_local_only_demo/models/sync_mode.dart';
+
+/// This schema design supports an online/local-only workflow by managing data
+/// across two versions of each table: one for local/offline use and one for
+/// online/synced use. This approach simplifies the handling of data in different
+/// connectivity states.
+///
+/// For local only, the views become:
+///   online_todos
+///   todos
+///   online_lists
+///   lists
+///
+/// - 'todos' and 'lists' refer to the local-only data.
+/// - 'online_todos' and 'online_lists' refer to the data that will be synced
+///    once online, making it clear that these are not currently synchronized.
```

Comment:
Thinking about it now I actually don't think the name `online_todos` makes it clear that these are not synced, they sound synced. What about `queued_todos`, `pending_upload_todos` or something like that?

Reply:
Just to confirm what we discussed - I like the new prefixes, and feel it makes it a little more more clear what's happening.